doc/tutorials/01_running_blink.md

```diff
@@ -18,23 +22,30 @@ Instructions
 
 1. **Compile Tock**. In the root of the Tock directory, compile the kernel for
 your hardware platform. You can find a list of boards by running `make list`.
+For example if your board is `imix then:
 
     ```bash
     cd tock
     make TOCK_BOARD=imix
     ```
 
+    If you have another board just replace imix with `<your-board>`
+
```

Comment:
I know this is a separate argument in a separate PR, but this might be a good place to explain that users can `cd` into the proper directory instead of using an environment variable.

Reply:
Well now you have to `cd` so this should be changed to `cd tock/boards/imix`.